gradle.properties

```diff
@@ -10,8 +10,9 @@
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
 org.gradle.jvmargs=-Xmx4g -XX:MaxPermSize=512M
+org.gradle.caching=true
```

Comment:
@CherryPerry Rather than updating the local `gradle.properties` that may have a negative impact on local builds, it might be a good idea to try using the approach that Google has taken on their Jetpack Compose dev challenge project.

https://github.com/android/android-dev-challenge-compose/blob/main/.github/workflows/Check.yaml

They have a different gradle.properties file specifically for building on CI that they use instead

Reply:
I can imagine how `org.gradle.jvmargs` or `org.gradle.workers.max` can affect the local builds, but not `org.gradle.caching` or `org.gradle.parallel`.

Reply:
fair enough. however perhaps we might want to make this change in another PR sometime in the future. Useful changes that you might want to put in this file is disabling Kotlin incremental processing and the Gradle daemon.